misinformation/test/test_multimodal_search.py

```diff
@@ -445,3 +445,113 @@ def test_parsing_images(
         multi_features_stacked,
     )
     cuda.empty_cache()
+
+
+@pytest.mark.long
+def test_itm():
+    test_my_dict = {
+        "IMG_2746": {
+            "filename": "../misinformation/test/data/IMG_2746.png",

```

Comment:
Can you use the `get_path` fixture for the file paths, so that this also works when the tests are initiated from a different directory; and also on Windows?
See [here](https://github.com/ssciwr/AMMICO/blob/3b1c3ef1edd5d64d57c5d1f897783209e30309fe/misinformation/test/test_faces.py#L8)